src/components/Card/Card.test.tsx

```
@@ -746,6 +746,17 @@ describe('CardStatus', () => {
     expect(screen.getByText('Errada')).toBeInTheDocument();
   });
 
+  it('should render with "Em branco" status', () => {
+    render(<CardStatus {...baseProps} status="unanswered" />);
+    expect(screen.getByText('Em branco')).toBeInTheDocument();
+  });
+
+  it('should render with "Em branco" status and label', () => {
+    render(<CardStatus {...baseProps} status="unanswered" label="Não respondida" />);
```

ESLint

[error] Replace `<CardStatus·{...baseProps}·status="unanswered"·label="Não·respondida"·/>` with `⏎······<CardStatus·{...baseProps}·status="unanswered"·label="Não·respondida"·/>⏎····`

(prettier/prettier)